### ISSUE TITLE
Garbled when I use rasax to add Chinese templates

### ISSUE THREAD

**Issue by xuexiaoao** [RasaHQ/rasa#4116]
**Rasa version**:
1.1.6
**Rasa X version** (if used & relevant):
0.19.5
**Python version**:
3.6.8
**Operating system** (windows, osx, ...):
windows
**Issue**:

Garbled when I use rasax to add Chinese templates,Unable to start when I restart rasax, prompt coding problem

**Error (including full traceback)**:
```
Traceback (most recent call last):
  File "c:\users\xuexi\anaconda3\envs\rasax\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\users\xuexi\anaconda3\envs\rasax\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\xuexi\Anaconda3\envs\rasax\Scripts\rasa.exe\__main__.py", line 9, in <module>
  File "c:\users\xuexi\anaconda3\envs\rasax\lib\site-packages\rasa\__main__.py", line 76, in main
    cmdline_arguments.func(cmdline_arguments)
  File "c:\users\xuexi\anaconda3\envs\rasax\lib\site-packages\rasa\cli\x.py", line 291, in rasa_x
    run_locally(args)
  File "c:\users\xuexi\anaconda3\envs\rasax\lib\site-packages\rasa\cli\x.py", line 315, in run_locally
    _validate_rasa_x_start(args, project_path)
  File "c:\users\xuexi\anaconda3\envs\rasax\lib\site-packages\rasa\cli\x.py", line 262, in _validate_rasa_x_start
    _validate_domain(os.path.join(project_path, DEFAULT_DOMAIN_PATH))
  File "c:\users\xuexi\anaconda3\envs\rasax\lib\site-packages\rasa\cli\x.py", line 275, in _validate_domain
    Domain.load(domain_path)
  File "c:\users\xuexi\anaconda3\envs\rasax\lib\site-packages\rasa\core\domain.py", line 72, in load
    other = cls.from_path(path, skill_imports)
  File "c:\users\xuexi\anaconda3\envs\rasax\lib\site-packages\rasa\core\domain.py", line 86, in from_path
    domain = cls.from_file(path)
  File "c:\users\xuexi\anaconda3\envs\rasax\lib\site-packages\rasa\core\domain.py", line 99, in from_file
    return cls.from_yaml(rasa.utils.io.read_file(path))
  File "c:\users\xuexi\anaconda3\envs\rasax\lib\site-packages\rasa\utils\io.py", line 130, in read_file
    return f.read()
  File "c:\users\xuexi\anaconda3\envs\rasax\lib\codecs.py", line 321, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xc4 in position 707: invalid continuation byte
```
![image](https://user-images.githubusercontent.com/18185523/62019796-0a54a700-b1f3-11e9-80ff-b600f02deb41.png)
![image](https://user-images.githubusercontent.com/18185523/62019813-20626780-b1f3-11e9-8d97-ac1243621a95.png)

**Command or request that led to error**:
```
rasa x
```

**Content of configuration file (config.yml)** (if relevant):
```yml

```

**Content of domain file (domain.yml)** (if relevant):
```yml

```

**Comment by msamogh** [comment]
Thanks for raising this issue, @Saladdin will get back to you about it soon.

**Comment by felaray** [comment]
same here.

![圖片](https://user-images.githubusercontent.com/5403865/63315040-44b9eb80-c33c-11e9-88fc-fd74e3afae2b.png)

**Comment by liujie19880815** [comment]
出现同样的问题了

**Comment by felaray** [comment]
nobody care this bug. no any response.

**Comment by Saladdin** [comment]
We are working on a fix for the issue. Will let you know once a patch is released 👍

**Comment by felaray** [comment]
thanks for help!

**Comment by felaray** [comment]
change setting in requirements.txt maybe solve it.
keras==2.2.4, numpy==1.16.5

**Comment by tmbo** [comment]
this should be fixed in 0.20.3
